src/main/java/com/dnd/runus/application/challenge/ChallengeService.java

```diff
@@ -48,9 +48,7 @@ public List<ChallengesResponse> getChallenges(long memberId) {
     }
 
     @Transactional
-    public ChallengeAchievementResponse save(long memberId, RunningRecord runningRecord, long challengeId) {
-        Member member =
-                memberRepository.findById(memberId).orElseThrow(() -> new NotFoundException(Member.class, memberId));
+    public ChallengeAchievementDto save(Member member, RunningRecord runningRecord, long challengeId) {
```

Comment:
`RunningRecord`의 필드에 `Member`가 있으니 member를 인자로 안 받아도 될 것 같아요
```suggestion
    public ChallengeAchievementDto save(RunningRecord runningRecord, long challengeId) {
```